app/components/StickyNote.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div
+    class="relative p-2 m-2 shadow-md h-60 w-60"
+    :style="{ backgroundColor: bgColor }"
+  >
+    <div
+      class="absolute top-0 left-0 px-2 py-1 text-xs text-black bg-white rounded-br"
+    >
+      {{ pinNumber }}
+    </div>
+    <textarea
+      v-model="modelValue"
+      placeholder="Add your note here..."
+      class="w-full h-full pt-6 text-sm bg-transparent border-none resize-none focus:outline-none"
+      @input="$emit('update:modelValue', $event.target.value)"
+    ></textarea>
+  </div>
+</template>
+
+<script>
+import { computed, defineComponent } from "vue";
+
+export default defineComponent({
+  props: ["modelValue", "color", "pinNumber"],
+  emits: ["update:modelValue"],
+  setup(props) {
+    const bgColor = computed(() => {
+      const colorMap = {
+        blue: "#0084C3",
+        pink: "#F1607D",
+        red: "#F64C00",
+        gold: "#CEA000",
+        green: "#02A856",
+        teal: "#03A58D",
+        purple: "#821EB1",
```

Comment:
I think these colors are also defined in the annote-marker thingy.  To avoid duplication of code, wee could move these to a definitions file in types?